lib/logdna/writer.go

```diff
@@ -127,10 +131,22 @@ func extractToken(u *url.URL) (token string, err error) {
 	return
 }
 
+func extractPEN(u *url.URL) (pen string) {
+	if len(pen) == 0 {
+		pen = "48950"
+	}
+
+	return
+}
```

Comment:
That's definitely my bad - no reason to override the pen anyway (not that you could do so from the URL regardless). I've moved the pen to be part of the tags without being separate variable, since it's only used once.